package.json

```diff
@@ -21,8 +21,8 @@
   },
   "peerDependencies": {
     "antd": "^3.2.1",
```

Comment:
Any issues if you update to the latest version of Ant? `3.16.6` as of now.

Reply:
Hi, yes, it was changing some element styles for us, hence I didn't want to force upgrade on users.